src/theme/index.tsx

```diff
@@ -390,15 +391,15 @@ export const ThemedGlobalStyle = createGlobalStyle`
   }
 
   .tf-v1-popover{
-    z-index: 999999 !important;
+    z-index: ${Z_INDEXS.ICON_SUPPORT} !important;
   }
 
   .tf-v1-popover-button {
     width: 48px !important;
     height: 48px !important;
     bottom: 84px !important;
     right: 16px !important;
-    background-color: ${({ theme }) => theme.primary} !important;
+    background-color: ${({ theme }) => theme.primary} !important;  
```

Comment:
Blank whitespace.

Reply:
What is the problem with that `Blank whitespace.`

Reply:
We should "prettier" the code before committing.

Reply:
got it